apps/client/src/features/rundown/event-block/EventBlock.utils.ts

```diff
@@ -18,24 +12,9 @@ export function formatDelay(timeStart: number, delay: number): string | undefine
   return `New start ${timeTag}`;
 }
 
-export function formatOverlap(timeStart: number, previousStart?: number, previousEnd?: number): string | undefined {
-  const noPreviousElement = previousEnd === undefined || previousStart === undefined;
-  if (noPreviousElement) return;
+export function formatOverlap(gap: MaybeNumber, isNextDay: boolean): string | undefined {
+  if (gap === null || gap === 0) return;
```

Comment:
suggestion: is it more expressive to say that we want gap to have a value?

```suggestion
  // we only care if gap has a value non-zero and non-nullish
  if (!gap) return;
```